cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1786,15 +1545,17 @@ private String insertNewWinnerRevision(SQLDatabase db,DocumentBody newWinner,
      *
      * @see #pickWinnerOfConflicts(long)
      */
-    private void setCurrent(SQLDatabase db, long sequence, boolean valueOfCurrent) {
+    public static void setCurrent(SQLDatabase db, long sequence, boolean valueOfCurrent) {
```

Comment:
I note that this hasn't been marked TODO and is not so obviously a callable because it doesn't already create an anonymous `SQLCallable` as in some other places, but I think this could also be moved into a callable, which I think suits it better than a public static method.